pkg/api/types.go

```diff
@@ -403,6 +404,13 @@ const (
 	PodUnknown PodPhase = "Unknown"
 )
 
+type PodCondition string
+
+// These are the valid service statuses of pods.
+const (
+	PodInService = "InService"
```

Comment:
I would like to follow the same pattern as NodeCondition:
https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/api/v1beta3/types.go#L793

```
type PodConditionKind string
const (
	PodLive PodConditionKind = "Live"
	PodReady PodConditionKind = "Ready"
)
```